tests/csv/3.0/fastq_pair.csv

```diff
@@ -1,3 +1,3 @@
 patient,gender,status,sample,lane,fastq_1,fastq_2
 test,XX,0,test,test_L1,https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/illumina/fastq/test_1.fastq.gz,https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/illumina/fastq/test_2.fastq.gz
-test,XX,1,test2,test2_L1,https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/illumina/fastq/test2_1.fastq.gz,https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/illumina/fastq/test2_2.fastq.gz
+test,XX,1,test2,test_L1,https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/illumina/fastq/test2_1.fastq.gz,https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/illumina/fastq/test2_2.fastq.gz
```

Comment:
could you please explain why this was changed to `test` rather than `test2`?

Reply:
Because the test_paired.csv was updated to have the same lane ID for both samples. That way I can test that the feature fixed in this PR is actually fixed.

Reply:
okay great!